### ISSUE TITLE
When start sublime text

### ISSUE THREAD

**Issue by 6aKa** [corneliusio/svelte-sublime#7]
```
... skip ...
ignored_packages: ["Vintage"]
... skip ...
reloading plugin Svelte.macros
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 125, in reload_plugin
    m = importlib.import_module(modulename)
  File "./python3.3/importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 1199, in load_module
    exec(compile(source, source_path, 'exec'), mod.__dict__)
  File "/Users/baka/Library/Application Support/Sublime Text 3/Installed Packages/Svelte.sublime-package/macros.py", line 1, in <module>
ImportError: No module named 'YAMLMacros'
```

**Comment by corneliusio** [comment]
This should be fixed in the next update. Plan to publish at some point today.

**Comment by corneliusio** [comment]
I've gone ahead and published the fix in 3.2.0.
